SDL/SDLJoystick.cpp

```diff
@@ -12,175 +16,174 @@ extern "C" {
 }
 
 SDLJoystick::SDLJoystick(bool init_SDL ): thread(NULL), running(true) {
-	if (init_SDL)
-	{
-		SDL_setenv("SDL_JOYSTICK_ALLOW_BACKGROUND_EVENTS", "1", 0);
-		SDL_Init(SDL_INIT_JOYSTICK | SDL_INIT_VIDEO);
+	if (init_SDL) {
+		SDL_SetHint(SDL_HINT_JOYSTICK_ALLOW_BACKGROUND_EVENTS, "1");
+		SDL_Init(SDL_INIT_JOYSTICK | SDL_INIT_VIDEO | SDL_INIT_GAMECONTROLLER);
+	}
+	
+	auto dbPath = File::GetExeDirectory() + "assets/gamecontrollerdb.txt";
```

Comment:
@xsacha will this work on Qt?  I think it bundles the assets into the executable?

-[Unknown]